network/onchain-discovery/src/test.rs

```diff
@@ -344,7 +351,13 @@ fn queries_storage_on_tick() {
         mut storage_query_ticker_tx,
         _peer_mgr_reqs_rx,
         conn_mgr_reqs_rx,
-    ) = setup_onchain_discovery(rt.handle().clone(), self_peer_id, role, storage_read_client);
+    ) = setup_onchain_discovery(
+        rt.handle().clone(),
+        self_peer_id,
+        role,
+        storage_read_client,
+        waypoint,
```

Comment:
Looking at this, it seems like we should have some means to specify a shared wrapper around storage or something that leverages waypoints so we don't splash waypoints all over the code.

Reply:
this is because on-chain discovery requires it